app/server.go

```diff
@@ -37,6 +37,7 @@ func main() {
 	r := mux.NewRouter()
 	r.HandleFunc("/karen/v1/users", env.PostUserHandler).Methods("POST")
 	r.HandleFunc("/karen/v1/users/auth", env.PostAuthHandler).Methods("POST")
+	r.HandleFunc("/karen/v1/users/self", env.GetUserHandler).Methods("GET")
```

Comment:
There should also be a separate `/karen/v1/users/{user-id}` handler. It can use the same `env.GetUserHandler` function as `/karen/v1/users/self` is using right now. The function will need to be updated to use the `{user-id}` from the URL, like [this](https://github.com/schramm-famm/ether/blob/7ac2c9f01455d149a050b71eefed6011b4e638d8/handlers/conversation_handlers.go#L103), if it is provided.